code/game/objects/items/devices/radio/radio.dm

```diff
@@ -120,6 +120,9 @@
 	if(type != /obj/item/radio)
 		return
 	AddElement(/datum/element/slapcrafting, string_list(list(/datum/crafting_recipe/improv_explosive)))
+	if(prob(0.5)) // Extremely rare chance to replace a normal radio with a toy one, because it's funny
+		new /obj/item/radio/toy(loc)
```

Comment:
dont' do it here, do it in the assistant job outfit datum in a post/preequip

Reply:
Part of me doesn't feel like that'd work great since my thought with it was these are really rare "collectors items"
meaning I think it's funnier if people stumble upon one in the wild rather than just spawning in with it, yknow?

Reply:
Fix the radio hard delete then

Reply:
should be fixed now, I think